cczoo/rats-tls/README.md

```diff
@@ -0,0 +1,176 @@
+# Building
+
+## Build Requirements
+
+- git
+- make
+- autoconf
+- libtool
+- gcc
+- g++ (ubuntu 18.04)
+- SGX driver, Intel SGX SDK & PSW: Please refer to this [guide](https://download.01.org/intel-sgx/sgx-linux/2.14/docs/Intel_SGX_SW_Installation_Guide_for_Linux.pdf) to install.
+- [SGX DCAP](https://github.com/intel/SGXDataCenterAttestationPrimitives): please download and install the packages from this [page](https://download.01.org/intel-sgx/sgx-dcap/#version#linux/distro).
+  - ubuntu 18.04: `libsgx-dcap-quote-verify-dev`, `libsgx-dcap-ql-dev`, `libsgx-uae-service`
+- For TDX, please see the README in TDX MVP Stack. You need to download the packages and following TDX_E2E_attestation_software_stack_Installation_README-dcap-2021XXXX.txt to do step 2 & step 3 to setup build and dependence libraries.
+
+## Build and Install
+
+Please follow the command to build RATS TLS from the latested source code on your system.
+
+1. Download the latest source code of RATS TLS
+
+```shell
+mkdir -p "$WORKSPACE"
+cd "$WORKSPACE"
+git clone https://github.com/inclavare-containers/rats-tls
+```
+
+2. Build and install RATS TLS
+
+```shell
+cd rats-tls
+cmake -DBUILD_SAMPLES=on -H. -Bbuild
+make -C build install
+```
+
+`{rats-tls-server,rats-tls-client}` will be installed to `/usr/share/rats-tls/samples/{rats-tls-server,rats-tls-client}` on your system. All instances are placed in `/opt/rats-tls/lib`.
+
+If you want to build instances related to sgx(sgx\_ecdsa, sgx\_ecdsa\_qve, sgx\_la), please type the following command.
+
+```shell
+cmake -DRATS_TLS_BUILD_MODE="sgx" -DBUILD_SAMPLES=on -H. -Bbuild
+make -C build install
+```
+
+If you want to run instances on libos occlum, please type the following command.
+
+```shell
+cmake -DRATS_TLS_BUILD_MODE="occlum" -DBUILD_SAMPLES=on -H. -Bbuild
+make -C build install
+```
+
+If you want to run TDX instances, please type the following command.
+```shell
+cmake -DRATS_TLS_BUILD_MODE="tdx" -DBUILD_SAMPLES=on -H. -Bbuild
+make -C build install
+```
+
+Note that [SGX LVI mitigation](https://software.intel.com/security-software-guidance/advisory-guidance/load-value-injection) is enabled by default. You can set macro `SGX_LVI_MITIGATION` to `0` to disable SGX LVI mitigation.
```

Comment:
I can't open this link. 
Is it still available? or changed to other links?

Reply:
please change it to https://www.intel.com/content/www/us/en/developer/articles/technical/software-security-guidance/technical-documentation/load-value-injection.html